SayTheirNames/Source/Dependency/Networking/NetworkRequestor.swift

```diff
@@ -40,24 +40,31 @@ final class NetworkRequestor: Dependency {
     public func fetchDecodable<T: Decodable>(_ url: String, completion: @escaping (Result<T, Swift.Error>) -> Swift.Void) {
         let request = self.session.request(url, headers: self.headers)
         request.responseDecodable(of: T.self, queue: self.concurrentQueue) { (response) in
-            DispatchQueue.mainAsync { completion(response.result.mapError({$0.swiftError})) }
+            DispatchQueue.mainAsync {             completion(response.result.swiftError)
+            }
         }
     }
     
     public func fetchData(_ url: String, completion: @escaping (Result<Data, Swift.Error>) -> Swift.Void) {
         let request = self.session.request(url, headers: self.headers)
         request.responseData(queue: self.concurrentQueue) { (response) in
-            completion(response.result.mapError({$0.swiftError}))
+            completion(response.result.swiftError)
         }
     }
     
     public func fetchJSON(_ url: String, completion: @escaping (Result<Any, Swift.Error>) -> Swift.Void) {
         let request = self.session.request(url, headers: self.headers)
         request.responseJSON(queue: self.concurrentQueue) { (response) in
-            completion(response.result.mapError({$0.swiftError}))
+            completion(response.result.swiftError)
         }
     }
 }
+// MARK: - Result + swiftError
+extension Result where Failure == AFError {
+    var swiftError: Result<Success, Error> {
+        return self.mapError{$0.swiftError}
+    }
+}
 // MARK: - AFError + swiftError
 extension AFError {
```

Comment:
Can we put this in the Units > Extension folder as `Result+STN.swift`?

Reply:
Whats going on with this?

Reply:
> Whats going on with this?

Do you mean the AFError extension should also go to the Units > Extension folder?

Reply:
Yeah lets throw that in there as well.

Reply:
> > Whats going on with this?
> 
> Do you mean the AFError extension should also go to the Units > Extension folder?

Indeed!

Reply:
Haha cool will throw it in another file now